File: Week3/BlackJackWeb/src/main/java/com/revature/blackjack/domain/Card.java
package com.revature.blackjack.domain;

public class Card {
	private int value;
	private String face;
	private String suit;
	
	public Card(int value, String face, String suit) {
		super();
		this.value = value;
		this.face = face;
		this.suit = suit;
	}

	public Card(int value)
	{
		this.value=value;
	}
	
	public int getValue() {
		return value;
	}
	public void setValue(int value) {
		this.value = value;
	}
	public String getFace() {
		return face;
	}
	public void setFace(String face) {
		this.face = face;
	}
	public String getSuit() {
		return suit;
	}
	public void setSuit(String suit) {
		this.suit = suit;
	}
	
	
}

File: Week4/blackjack-ng/src/app/types/Card.ts
export class Card {

    private value: number;

    private suit: string;

    public getValue(): number {
        return this.value;
    }

    public setValue(value: number): void {
        this.value = value;
    }

    public getSuit(): string {
        return this.suit;
    }

    public setSuit(suit: string): void {
        this.suit = suit;
    }

}